_locales/en/messages.json

```diff
@@ -1850,6 +1850,22 @@
     "message": "Hide menu bar",
     "description": "Label text for menu bar visibility setting"
   },
+  "stayInBackground": {
+    "message": "Stay in Background",
```

Comment:
"Stay in Background" reads a little clunky to me; it suggests the app is already in the background and it should stay there.  I think I more commonly see something along the lines of "Keep running in the background".  This suggests the app is currently running, and it should stay running, but in the background instead.  I'm not sure if the Signal team has someone on hand for technical language in the UI.